### PR TITLE
Add Sketch assets

### LINKED ISSUE #417: Sketch files
Hi, do we have available Sketch files for the components please?

### PR DESCRIPTION
**Pull request by llorca**
fixes #417

+ a new simple "Resources" page in the docs

### CODE REVIEW

File: packages/core/src/_resources.scss [hunk 1]
```diff
@@ -0,0 +1,27 @@
+// Copyright 2015 Palantir Technologies, Inc. All rights reserved.
```
Comment (giladgray) [inline packages/core/src/_resources.scss]: 2017

File: packages/core/src/_resources.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Weight: 10
+
+Styleguide resources
+*/
+
+/*
+Sketch assets
+
+<a href="https://github.com/palantir/blueprint/tree/master/resources/sketch/Blueprint%20Kit.sketch" class="docs-asset">
```
Comment (giladgray) [inline packages/core/src/_resources.scss]: - `target="_blank"` on both links
- put `class` first (alphabetize)

File: packages/core/src/_resources.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+/*
+Resources
+
+Weight: 10
+
+Styleguide resources
+*/
+
+/*
+Sketch assets
```
Comment (giladgray) [inline packages/core/src/_resources.scss]: is there anything to say about these files? some intro to using them / what to expect?

Comment (llorca) [inline packages/core/src/_resources.scss]: There's actually a description _inside_ the file

**Comment by NickyYo** [comment]
@giladgray @llorca fantastic! thank you for this, great project! expect some pull requests from me in the future.

**Comment by llorca** [comment]
@NickyYo Thanks! Looking forward to it 👍  🎉

**Comment by minasyanlv** [comment]
Hi, can't download the palette, or do I do something wrong? How to use it?

**Pull request merged**